packages/kotti-ui/source/kotti-comment/index.ts

```diff
@@ -4,20 +4,29 @@ import { attachMeta, makeInstallable } from '../utilities'
 
 import KtCommentVue from './KtComment.vue'
 import KtCommentInputVue from './KtCommentInput.vue'
+import { KottiComment, KottiCommentInput } from './types'
 
-const META: Kotti.Meta = {
+const META: Omit<Kotti.Meta, 'typeScript'> = {
 	addedVersion: '0.0.1',
 	deprecated: null,
 	designs: {
 		type: MetaDesignType.FIGMA,
 		url: 'https://www.figma.com/file/0yFVivSWXgFf2ddEF92zkf/Kotti-Design-System?node-id=133%3A2594',
 	},
 	slots: {},
-	typeScript: null,
 }
 
-export const KtComment = attachMeta(makeInstallable(KtCommentVue), META)
-export const KtCommentInput = attachMeta(
-	makeInstallable(KtCommentInputVue),
-	META,
-)
+export const KtComment = attachMeta(makeInstallable(KtCommentVue), {
+	...META,
+	typeScript: {
+		namespace: 'Kotti.Comment',
```

Comment:
I think those will add the `TS` labels to the documentation. I can set these to `null` for now and set again when components are refactored into composition api in a future step.

Reply:
Yes, it does, but the most important part about the TS label is that it exposes its props as a TS namespace anyway, so I guess it’s fine, especially given that you’re also refactoring the rest of this in the near future